python/sidecar/ingest.py

```diff
@@ -4,21 +4,29 @@
 import sys
 from dataclasses import asdict
 from pathlib import Path
-from typing import List
+from typing import Dict, List
 
 import grobid_tei_xml
 from grobid_client.grobid_client import GrobidClient
 
 from .shared import HiddenPrints, chunk_text, get_filename_md5
 from .typing import Author, Reference
 
+logging.root.setLevel(logging.NOTSET)
+
 logger = logging.getLogger(__name__)
-logger.setLevel(logging.INFO)
+handler = logging.FileHandler(
+    os.path.join(
+        os.environ.get("SIDECAR_LOG_DIR", "/tmp"), "refstudio-sidecar.log"
```

Comment:
shouldn't we call it `refstudio-sidecar-ingest.log` instead?

Reply:
My intention is to have one unified log for the sidecar that includes both ingest and interactions.